modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -5,7 +5,8 @@
 // * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
 // * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
 // * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
-// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.
```

Comment:
```suggestion
// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
```
Trivial, but fixing it will help avoid merge conflicts and keep the branch in sync with `main`

Reply:
Thanks. Changes made